xarray/tests/test_dask.py

```diff
@@ -292,6 +293,26 @@ def test_persist(self):
         self.assertLazyAndAllClose(u + 1, v)
         self.assertLazyAndAllClose(u + 1, v2)
 
+    @requires_pint
+    def test_tokenize_duck_dask_array(self):
+        import pint
+
+        pytest.mark.skipif(
+            pint.__version__ < "0.15",
+            reason="Dask tokenize is not implemented for Pint < 0.15",
+        )
```

Comment:
does this actually work (same also for the other test)? If not, we might want to switch to `pytest.skip` (possibly in combination with a normal branch if `pytest.skipif` does not exist). Also, I'm not sure the string comparison is guaranteed to work; we might want to pass both to `LooseVersion` before comparing (even better would be `packaging.version.parse` but that would be a new dependency).

Reply:
No, it doesn't, good catch. What about requiring the minimum version to be 0.15 for `has_pint` and `requires_pint`? Currently these tests are the only ones using the `requires_pint` wrapper, but it could be a source of confusion if it is used elsewhere in the future.  Thoughts?

Reply:
If you need a specific minimum version, we could create a new set, kind of like what already exists for `cftime`: https://github.com/pydata/xarray/blob/55480de69096cc5ae003f639c2c953066e829120/xarray/tests/__init__.py#L68-L69

Reply:
Sounds good, I'll do something similar for `pint`.